src/tests/fuzzing/fuzz_pkcs15init.c

```diff
@@ -122,7 +122,7 @@ void do_init_app(struct sc_profile *profile, struct sc_pkcs15_card *p15card, sc_
                  unsigned char *so_pin, unsigned char *so_puk)
 {
     struct sc_pkcs15init_initargs init_args;
-    sc_pkcs15_auth_info_t         info;
+    sc_pkcs15_auth_info_t info = {0};
```

Comment:
below, we have `memset(&info, 0, sizeof(info));` shouldn't this be equivalent?

Reply:
indeed, it is covered by b90191e6c53aba954370e1db9a09a843124c24f5